webapp/lib/view_model.dart

```diff
@@ -134,9 +134,11 @@ class CodeSelector {
   static CodeSelector get activeCodeSelector => _activeCodeSelector;
   static set activeCodeSelector(CodeSelector activeCodeSelector) {
     _activeCodeSelector?.viewElement?.classes?.toggle('active', false);
+    _activeCodeSelector?.viewElement?.parent?.parent?.classes?.toggle('active', false);
     // Focus on the new code selector
     _activeCodeSelector = activeCodeSelector;
     _activeCodeSelector.viewElement.classes.toggle('active', true);
+    _activeCodeSelector?.viewElement?.parent?.parent?.classes?.toggle('active', true);
```

Comment:
I don't think we need null-aware operators here, though if you want, we could check before ln 139 if the new code selector is null and throw an error, as that should never happen.